src/univariate/continuous/biweight.jl

```diff
@@ -54,16 +54,16 @@ end
 
 @quantile_newton Biweight
 
-function mgf(d::Biweight{T}, t::Real) where T<:Real
-    a = d.σ*t
+function mgf(d::Biweight, t::Number)
+    a = d.σ * t
     a2 = a^2
-    a == 0 ? one(T) :
+    iszero(a) ? one(Base.promote_typeof(a, d.μ)) :
```

Comment:
This is still type unstable (and similar cases below) if e.g. all involved values are `Int`s or `Rational`s. Maybe it's easiest to just compute the result and return `one(result)` in this special case.